src/main/java/com/ericsson/ei/frontend/utils/BackEndInformationControllerUtils.java

```diff
@@ -31,16 +31,14 @@
 import org.springframework.stereotype.Component;
 
 import com.google.gson.JsonArray;
-import com.google.gson.JsonObject;
-import com.google.gson.JsonParser;
 
 @Component
 public class BackEndInformationControllerUtils {
```

Comment:
Please rename this class to for example BackEndInstanceControllerImpl.
Is this class needed? it has any logic?

Reply:
It's not really an "impl" class as we don't implement an interface etc. I think that would be a confusion. But i agree that we should change its name.